ipaplatform/fedora/paths.py

```diff
@@ -26,13 +26,21 @@
 from __future__ import absolute_import
 
 from ipaplatform.redhat.paths import RedHatPathNamespace
+from ipaplatform.osinfo import osinfo
+
+# Fedora 28 and earlier use /etc/sysconfig/nfs
+# Fedora 30 and later use /etc/nfs.conf
+# Fedora 29 has both
+HAS_NFS_CONF = osinfo.id == 'fedora' and int(float(osinfo.version_id)) >= 30
```

Comment:
Why can't you import `HAS_NFS_CONF` from the constants instead? It is better to have the check in a single place.

Reply:
Done, thanks @abbra